### ISSUE TITLE
Fix publish coverage error

### ISSUE THREAD

**Issue by svenvanderburg** [fair-workflows/nanopub#95]
We had this error:
```
coveralls.exception.CoverallsException: Running on Github Actions but GITHUB_TOKEN is not set. Add "env: GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}" to your step config.
```
on #94

**Comment by vemonet** [comment]
Not sure how your test coverage work, but the error message seems not to be pointing to the right issue

Usually after doing the tests it will should try to publish the test coverage to the website you use (coveralls here apparently)

The action workflow:

```yaml
env:
          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
```

I did not set the secret `COVERALLS_REPO_TOKEN` (and should not, it is for your repo only)

What you can do to make it cleaner probably is to add a `if` for the coverall publishing step, so that the step is run only if the `secrets.COVERALLS_REPO_TOKEN` is set

**Comment by svenvanderburg** [comment]
Fixed by #98

**Comment by svenvanderburg** [comment]
Thanks a lot @vemonet 👍
